src/index.ts

```diff
@@ -315,7 +315,9 @@ export class Probot {
   }
 
   public start() {
-    this.log.info(`Running Probot v${this.version}`);
+    this.log.info(
+      `Running Probot v${this.version} - Node.js ${process.version}`
```

Comment:
```suggestion
      `Running Probot v${this.version} (Node.js: v${process.version})`
```

Reply:
Why not, even better

Reply:
Argh, wait, this will print two times the leading `v` for Node.js